cluster/selfplay/cc_wrapper.sh

```diff
@@ -40,8 +40,8 @@ then
     --model=$NAME \
     --num_readouts=800 \
     --mode=selfplay \
-    --resign_threshold=0.91 \
-    --disable_resign_pct=0.2 \
+    --resign_threshold=0.99 \
+    --disable_resign_pct=0.1 \
```

Comment:
is this a percent or fraction?

Reply:
it's a fraction o.o  And actually, i didn't check that these flags all still exist this way.

Reply:
actually this is true for all the flags. you want s/pct/amt?  something else?

Reply:
s/pct/frac I think